sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -487,3 +490,53 @@ def __init__(self):
         self.signed_service = None
         self.signed_version = None
         self.value = None
+
+class TokenAudience(Protocol):
+    """
+    Protocol that defines what calling functions and attributes should be defined for an Azure Active
+    Directory audience token.
+
+    :param str value: The Azure Active Directory audience to use when forming authorization scopes.
+        For the Language service, this value corresponds to a URL that identifies the Azure cloud
+        where the resource is located. For more information please take a look at:
+        https://learn.microsoft.com/azure/storage/blobs/authorize-access-azure-active-directory
+        Please use one of the static constant members over creating a custom value unless you have
+        a specific scenario for doing so.
```

Comment:
> The audience to use when requesting tokens for Azure Active Directory authentication. For more information please take a look at: https://learn.microsoft.com/azure/storage/blobs/authorize-access-azure-active-directory. Please use one of the provided classmethods over creating a custom value unless you have a specific scenario for doing so.

Reply:
I actually "borrowed" this from Amanda, so you can blame Amanda 😛